src/main/java/org/opentripplanner/graph_builder/module/AddTransitModelEntitiesToGraph.java

```diff
@@ -196,13 +220,21 @@ private void createPathwayEdgesAndAddThemToGraph(Graph graph) {
                     createElevatorEdgesAndAddThemToGraph(graph, pathway, fromVertex, toVertex);
                 }
                 else {
+
+                    // the GTFS spec allows you to define a pathway which has neither traversal time, distance
+                    // nor steps. This would lead to traversal costs of 0, so we compute the distance from the
+                    // vertices as fallback.
+                    double distance = Optional.of(pathway.getLength())
+                            .filter(l -> l <= 0)
+                            .orElseGet(() -> distance(fromVertex.getCoordinate(), toVertex.getCoordinate()));
```

Comment:
The condition is reversed:
```suggestion
                    double distance = Optional.of(pathway.getLength())
                            .filter(l -> l > 0)
                            .orElseGet(() -> distance(fromVertex.getCoordinate(), toVertex.getCoordinate()));
```

Reply:
An embarrassing mistake! Thanks for spotting.